### PR TITLE
Print parameters as YAML in parameters_only_entry_point().

### LINKED ISSUE #75: Print parameters as YAML in parameters_only_entry_point()
`parameters_only_entry_point()` currently prints its parameters as a string. It would be more readable if it printed these as YAML (since that's how the parameter files are formatted).

### PR DESCRIPTION
**Pull request by spigo900**
Resolves #75.

### CODE REVIEW

**Comment by spigo900** [comment]
I'm not sure if the current output is what we want. Here's what it looks like:

```
state: !!python/tuple
- !!python/object/apply:immutablecollections._immutabledict.immutabledict
  - !!python/tuple
    - !!python/tuple
      - input_zip
      - E:\shared\Coding\ISI\aida-2019\/data/rpi_inputs/RPI_TA1_0530_clean.zip
    - !!python/tuple
      - output_zip
      - E:\shared\Coding\ISI\aida-2019\/data/outputs/2019-03-corpus/2019-03-corpus.coref_enriched.zip
    - !!python/tuple
      - corpus_zip
      - E:\shared\Coding\ISI\aida-2019\/data/corpora/dryrun-2019-03.text.zip
```

File: vistautils/parameters_only_entrypoint.py [hunk 1]
```diff
@@ -26,7 +28,7 @@ def parameters_only_entry_point(
     if len(sys.argv) == 2:
         params = YAMLParametersLoader().load(sys.argv[1])
         configure_logging_from(params)
-        log.info("Ran with parameters:\n%s", params)
+        log.info("Ran with parameters:\n%s", yaml.dump(params))
```
Comment (rgabbard) [inline vistautils/parameters_only_entrypoint.py]: A few things to make the output prettier:

* I think we want to use `default_flow_style=False` (see first FAQ at https://pyyaml.org/wiki/PyYAMLDocumentation)
* I think we want to have a custom representer to make the output look closer to the usual input, using `yaml.add_representer`

Comment (spigo900) [reply vistautils/parameters_only_entrypoint.py]: Alright, I'll look at doing that.

Comment (spigo900) [reply vistautils/parameters_only_entrypoint.py]: @rgabbard I set up a representer and it's looking much better now:

```
INFO:vistautils.parameters_only_entrypoint:Ran with parameters:
!<Parameters>
corpus_zip: E:\shared\Coding\ISI\2019-aida\/data/corpora/dryrun-2019-03.text.zip
input_zip: E:\shared\Coding\ISI\2019-aida\/data/rpi_inputs/RPI_TA1_0531_clean.zip
output_zip: E:\shared\Coding\ISI\2019-aida\/data/outputs/2019-03-corpus/2019-03-corpus.coref_enriched.zip
```

(If you prefer to have everything quoted, that's also possible.)

I looked at getting it to use double quotes for parameter values (and leave the keys unquoted), but it doesn't look like YAML provides a nice way to do that. If you prefer to get it in that format, I did find a slightly hacky way to do it by reaching inside the `represent_mapping()` return value. It looks like this:

```
def new_representer(dumper, params):
    mapping_node = dumper.represent_mapping("Parameters", params.as_mapping())
    for _, value in mapping_node.value:
        value.style = '"'
    return mapping_node
```

Or we can live with all of the keys/values being quoted (or all unquoted).

Comment (rgabbard) [reply vistautils/parameters_only_entrypoint.py]: I think leaving them all unquoted is fine.  Is there a way to ditch the `!<Parameters>` at the beginning?

Comment (rgabbard) [reply vistautils/parameters_only_entrypoint.py]: (it is possible we need to convert it all into nested dicts and then `dump` that)

Comment (spigo900) [reply vistautils/parameters_only_entrypoint.py]: @rgabbard It took a bit of looking, but I found a way to get rid of the `!<Parameters>`. Changing the tag from "Parameters" to "tag:yaml.org,2002:map" gets rid of it. I tested and it seems to also work when you have nested dictionary parameters.

**Comment by spigo900** [comment]
Ready for re-review.

**Pull request merged**